test/TablelandTables.ts

```diff
@@ -0,0 +1,200 @@
+import { SignerWithAddress } from "@nomiclabs/hardhat-ethers/signers"
+import chai from "chai"
+import chaiAsPromised from "chai-as-promised"
+import { BigNumber } from "ethers"
+// @ts-ignore-start
```

Comment:
What's going on here? Why the ts-ignore?

Reply:
it's odd... whenever i `import { ethers } from "hardhat"` i get this error

https://github.com/NomicFoundation/hardhat/issues/1627

but everything works fine, include using TS_NODE_FILES. i tried everything in that and other issues to get the red squiggle to go away.... maybe you have some magic tricks

Reply:
oh, figured it out. ethers had to be in dev dependencies. must be something that changed in recent version